src/plugin/notification/interfaces.ts

```diff
@@ -1,8 +1,9 @@
 export type NotificationPluginConfiguration = {
   name?: string;
   type: string;
-  public: boolean;
```

Comment:
I would like to keep `public`. By default it would be true and produces classic notification. If set to `false` it would produce notifications specific for the baker like fee collected and similar data which may have sensitive aspects. 

Alternatively we can use `private` - `true`.